docs/05-command-line.md

```diff
@@ -218,13 +218,13 @@ When running a file with and without line numbers, line numbers take precedence.
 
 ## Resetting AVA's cache
 
-AVA may cache certain files, especially when you use our [`@ava/babel`](https://github.com/avajs/babel) provider. If it seems like your latest changes aren't being picked up by AVA you can reset the cache by running:
+AVA itself does not cache files unless used with our [`@ava/babel`](https://github.com/avajs/babel) provider. However, if it seems like your latest changes aren't being picked up by AVA you can try resetting the cache by running:
```

Comment:
```suggestion
AVA itself does not cache files unless used with our [`@ava/babel`](https://github.com/avajs/babel) provider. If it seems like your latest changes aren't being picked up by AVA you can try resetting the cache by running:
```

`However` makes it seem like this does apply even if you don't use `@ava/babel`.